tests/integration/targets/gitlab_group_variable/tasks/main.yml

```diff
@@ -575,11 +575,11 @@
     purge: True
   register: gitlab_group_variable_state
 
-- name: check that no variables are untoucheded state must be changed
+- name: check that no variables are untouched state must be changed
   assert:
     that:
       - not gitlab_group_variable_state.changed
```

Comment:
@scodeman and the state is also changed ;)

Reply:
```suggestion
      - gitlab_group_variable_state.changed
```

Reply:
Not sure how this got misaligned with my clone but here it is fixed